### ISSUE TITLE
Bug Report: The system image does not run on Orange Pi 5 Plus

### ISSUE THREAD

**Issue by grhgrmgrhrm** [Joshua-Riek/ubuntu-rockchip#1126]
### What happened?

Hi! I downloaded different versions of the operating system for OrangePi 5 Plus and tried to burn them both to an SD card and a USB flash drive and also directly burn them to eMMC. The system does not start on OrangePi 5 Plus. 
I've tried different flash drives, different versions of the system - 22.04 , 24.04, 24.19 - and desktop and server versions. 
The system does not start in any way. At the same time, a competitive system (called Arm...) it starts on my board the first time. Also, I check the checksums of the image before writing it to a flash drive. The operating system I'm trying to burn the image from is Windows. Recording utilities - balenaEtcher, Rufus, Universal Usb installer.
Please take a look - maybe your images for this board are not being assembled correctly?

### Kernel version

6.1.0-1020-rockchip

### SBC model

Orange Pi 5 Plus

### What operating system are you seeing this problem on?

Ubuntu 22.04 LTS (Jammy Jellyfish), Ubuntu 24.04 LTS (Noble Nombat)

**Comment by dgdonovan** [comment]
See https://github.com/Joshua-Riek/ubuntu-rockchip/discussions/1104

**Comment by kaosbeat** [comment]
also, I have no problem at all running the 24.04 on the OPi5-plus, what instructions did you follow to get it going? can you SSH into the system? maybe no HDMI but it still boots?

**Comment by grhgrmgrhrm** [comment]
> also, I have no problem at all running the 24.04 on the OPi5-plus, what instructions did you follow to get it going? can you SSH into the system? maybe no HDMI but it still boots?

First of all, all attempts to start the board were through connection to a TV via HDMI port. Also I tried to follow this instruction - https://github.com/Joshua-Riek/ubuntu-rockchip?tab=readme-ov-file#installation
0) When recording an image using Belena, a verification error popped up after the recording itself. The image was downloaded as well as the url for downloading through the program. The image was additionally checked against checksums, but Belena never successfully verified the image recording. Tried to write with other utilities - ok, but:
1) SD card did not boot, black screen
2) from USB did not boot, black screen
3) I installed the system through the manufacturer's instructions, using the RKDevTool utility directly on the emmc memory - the same thing, black screen
4) I also tried to boot from the SD card to Armbian and then download the Ubuntu image from there to the nvme disk, which I then copied to emmc via the command line - the result of the download is a black screen.


The board seemed to boot itself, green LED was blinking and red LED was lit in parallel, but nothing was displayed on the screen. On the other hand, I performed exactly the same actions for Armbian - it starts up from the first time.

**Comment by ctshieh** [comment]
Same here

**Comment by devdotnetorg** [comment]
@grhgrmgrhrm I use Armbian and Joshua-Riek images on SD card and SSD disk, everything is OK. I write images using balenaEtcher-Portable-1.18.11.exe and 16 GB microSD. Most likely you have a problem with writing the image to disk.

**Comment by ctshieh** [comment]
Thank you very much, I shall try when getting a new card, thanks again

**Comment by ctshieh** [comment]
I  had tried  several times and failed again and again.
I can install ubuntu 22.04 without any problem but I always got black screen while booting with ubuntu 24.xx.
I wonder if the problem is due to the board itself or the image.  The ubuntu 24.xx image was written  to  a SD card under Mac OS system (by balenaEtcher).

**Comment by MarcAndreWyss** [comment]
If you are using 24.10 HDMI wil not work. See here: https://github.com/Joshua-Riek/ubuntu-rockchip/wiki/Ubuntu-24.10

**Comment by ctshieh** [comment]
Thanks very much. It works good under Ubuntu 24.04.1 . Thanks again.

**Comment by ewaldc** [comment]
Several versions of Balena Etcher on Windows/Mac have issues dealing with gzip and xz compressed archives (see [here](https://github.com/balena-io/etcher/issues/4064)).  Since all assets on this site are all xz compressed, there is a high chance of hitting this problem. Hence, as others have said, use other decompression tools e.g. `7z` to decompress the assets first: e.g. "ubuntu-24.04-preinstalled-desktop-arm64-orangepi-5-plus.img.xz" into "ubuntu-24.04-preinstalled-desktop-arm64-orangepi-5-plus.img".  Flashing the .img file will succeed (given a good SSD card), including verification.

**Comment by grhgrmgrhrm** [comment]
Hello everybody! The system started from the microsd card - I installed the unpacked version on 22.04 via Belena. And then I installed it on eMMC without any problems. Thank you all again, and special thanks. - @devdotnetorg @ctshieh @ewaldc

**Comment by grhgrmgrhrm** [comment]
It is unknown what happened after many unsuccessful attempts, but I wrote a system image to the same microSD card from the same computer and it suddenly worked. Colleagues said - Christmas magic. What I wish you all!

**Comment by lonely-soul1234** [comment]
> > also, I have no problem at all running the 24.04 on the OPi5-plus, what instructions did you follow to get it going? can you SSH into the system? maybe no HDMI but it still boots?
> 
> First of all, all attempts to start the board were through connection to a TV via HDMI port. Also I tried to follow this instruction - https://github.com/Joshua-Riek/ubuntu-rockchip?tab=readme-ov-file#installation 0) When recording an image using Belena, a verification error popped up after the recording itself. The image was downloaded as well as the url for downloading through the program. The image was additionally checked against checksums, but Belena never successfully verified the image recording. Tried to write with other utilities - ok, but:
> 
> 1. SD card did not boot, black screen
> 2. from USB did not boot, black screen
> 3. I installed the system through the manufacturer's instructions, using the RKDevTool utility directly on the emmc memory - the same thing, black screen
> 4. I also tried to boot from the SD card to Armbian and then download the Ubuntu image from there to the nvme disk, which I then copied to emmc via the command line - the result of the download is a black screen.
> 
> The board seemed to boot itself, green LED was blinking and red LED was lit in parallel, but nothing was displayed on the screen. On the other hand, I performed exactly the same actions for Armbian - it starts up from the first time.

I had same issue with TF card, balena flashes .xz file cause an verification error and can not load machine. but I unxz the file to an .image file, it works fine. The ubuntu starts up as excepted.
